### PR TITLE
nflverse-data URL updates

### PR DESCRIPTION
**Pull request by tanho63**

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -44,3 +44,13 @@ NULL
 `%c%` <- function(x,y){
   ifelse(!is.na(x),x,y)
 }
+
+#' @export
+#' @noRd
+print.nflverse_data <- function(x,...){
+
+  cli::cli_rule("{.emph nflverse {attr(x,'nflverse_type')}}")
+  cli::cli_alert_info("Data updated: {.val {attr(x,'nflverse_timestamp')}}")
```
Comment (mrcaseb) [inline R/utils.R:53]: My suggestion for this that prints the update time in the current systems' timezone
```r
cli::cli_alert_info(
  "Data updated: {.code {format(attr(x,'nflverse_timestamp'), 
   tz = Sys.timezone(), usetz = TRUE)}}"
)
```

Comment (tanho63) [reply R/utils.R]: Great idea, going with 
```r
  if(!is.null(attr(x,'nflverse_timestamp'))) {
    cli::cli_alert_info(
      "Data updated: {.field {format(attr(x,'nflverse_timestamp'), tz = Sys.timezone(), usetz = TRUE)}}"
      )
  }
  ```

Comment (tanho63) [reply R/utils.R]: image

Comment (mrcaseb) [reply R/utils.R]: Sounds good!

**Review by mrcaseb — APPROVED** [review]
I think NEWS.md needs an update. The rest lgtm

**Pull request merged**